src/commands/login.ts

```diff
@@ -49,7 +59,10 @@ export const login = async (
   await writeIdentity(orgWithSlug, tokenResponse);
 
   // validate auth
-  if (!options?.skipAuthenticate) await authenticate({ noRefresh: true });
+  if (!options?.skipAuthenticate) {
+    const identity = await loadCredentials({ noRefresh: true });
+    await authenticateToFirebase(identity);
```

Comment:
It's because authenticate now saves the config & initializes firebase. I would like to rename this method now, but couldn't think of a better name.

And there used to be a circular call chain, which I am trying to break:

authenticate -> loadCredentials -> login -> authenticate